07-sampling.Rmd

```diff
@@ -1064,33 +1062,245 @@ Comment on the representativeness of the following *sampling methodologies*:
 ```
 
 
-## Central Limit Theorem {#sampling-conclusion-central-limit-theorem}
+## A theoretical approach and the Central Limit Theorem
 
-This chapter began with (virtual) access to a large bowl of balls (our population) and a desire to figure out the proportion of red balls. Despite having access to this population, in reality, you almost **never** will have **access to the population,** either because the population is too large, ever changing, or too expensive to take a census of. Accepting this reality means accepting that we need to use _statistical inference._ 
+We now introduce a simple theoretical approach to understanding some important properties of the sample proportion. To do this we modify the bowl activity slightly. Instead of using a shovel to select all 25 balls at once, we randomly select one ball at a time. If the ball is red we call it a success and record a 1 (one); if it is not red we call it a failure and record a 0 (zero). Then, we return the ball to the bowl so the proportion of red balls in the bowl doesn't change. 
+This process is called a trial or a Bernoulli trial in honor of Jacob Bernoulli, a 17th-century mathematician who used it already back in those times. 
+Getting a sample of 25 balls is running 25 trials and obtaining 25 numbers, ones or zeros, representing whether we have observed red balls or not on each trial, respectively. 
+The average of these 25 numbers is the sample proportion. 
```

Comment:
@avaldivi6 I think laying out an example here to show that the average of these 25 numbers will result in the sample proportion would be helpful.

Reply:
You do introduce what this looks like with terminology just after this, but we've usually followed laying things out without the terminology and then assigning terminology to reduce cognitive load.